winterfell/middlewares/redis-session.js

```diff
@@ -22,8 +22,8 @@ module.exports = function (app) {
       store: new redisStore({
         host: host,
         port: port,
-        client: redisClient,
-        ttl: Constants.SESSION_EXPIRE_TIME}), // expires after 20 minutes
+        client: redisClient}), 
+      cookie: {expires: new Date(Date.now() + Constants.SESSION_EXPIRE_TIME)}, // expires after 20 minutes
```

Comment:
Yes exactly when the user hits the login link, express-session tells the browser to set an http only cookie. The http only cookie cannot be accessed in any way by the client side javascript, which makes it more robust to XSS attacks. The server will also set this expiration date on the cookie. When the server sees the cookie is older than the expiration date, the user will be automatically logged out. All of the real authentication stuff happens on the server side.